devices.js

```diff
@@ -10228,6 +10228,25 @@ const devices = [
             await configureReporting.onOff(endpoint);
         },
     },
+    {
+        zigbeeModel: ['TS110F'],
```

Comment:
Hi!
Could you, please, follow-up with a change that replaces zigbeeModel with something like:
`        fingerprint: [{modelID: 'TS110F', manufacturerName: '_TYZB01_qezuin6k'}],`
(the fingerprint above is for 1-gang dimmer, which has the same model ID). You can find the manufacturerName in the logs when pairing:
```
debug 2020-08-25 18:36:09: Received Zigbee message from '0xbc33acfffe6d81d6', type 'readResponse', cluster 'genBasic', data '{"modelId":"TS110F"}' from endpoint 1 with groupID 0
debug 2020-08-25 18:36:09: Received Zigbee message from '0xbc33acfffe6d81d6', type 'readResponse', cluster 'genBasic', data '{"manufacturerName":"_TYZB01_qezuin6k"}' from endpoint 1 with groupID 0
```